app/assets/stylesheets/components/header.scss

```diff
@@ -124,22 +124,29 @@
 
 .skip-content-link {
   position: absolute;
-  top: 0;
   left: 50%;
-  transform: translate(-50%, -100px);
   background: var(--body-color);
   color: var(--body-color-inverted);
   z-index: var(--z-popover);
   padding: var(--su-4);
   pointer-events: none;
-  opacity: 0;
   border-radius: var(--radius);
-  outline: 0;
+  height: var(--header-height);
+  transform: translate(-50%, calc(-1 * var(--header-height)));
+  transition: transform 0.1s ease-in;
 
-  &:focus {
+  &:focus,
+  &:focus-within {
     pointer-events: auto;
-    opacity: 1;
-    transform: translate(-50%, var(--header-height) / 2);
+    transform: translate(-50%, 0);
+    outline: none;
+    border: 2px solid var(--header-button-focus-color);
+  }
+
+  @media (prefers-reduced-motion: reduce) {
+    &.skip-content-link {
+      transition-duration: 0;
+    }
```

Comment:
We don't need this since we have this in our [reset](https://github.com/forem/forem/blob/master/app/assets/stylesheets/base/reset.scss#L155-L163) by default 👍 .